wrangler-proto/src/main/java/io/cdap/wrangler/proto/recipe/v2/RecipeCreationRequest.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.wrangler.proto.recipe.v2;
+
+import java.util.List;
+
+/**
+ * Creation request for a Recipe
+ */
+public class RecipeCreationRequest {
+  private final String recipeName;
+  private final String description;
+  private final List<String> directives;
+
+  public RecipeCreationRequest(String recipeName, String description, List<String> directives) {
```

Comment:
Namespace should also be part of this.

Reply:
Not necessary as namespace is passed in URL as a path parameter